webpack.config.js

```diff
@@ -1,20 +1,21 @@
-const webpack = require('webpack'),
-      path = require('path'),
-      I18nPlugin = require('i18n-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin'),
+    I18nPlugin = require('i18n-webpack-plugin'),
+    path = require('path'),
+    webpack = require('webpack');
 
 const LANGUAGES = [
     'en-US',
     'es-MX',
     'pt-BR',
 ];
 
-const distFolder = path.join(__dirname, 'dist');
+// const distFolder = path.join(__dirname, 'dist');
```

Comment:
If we are going to get rid of the variable, we shouldn't comment it out but delete its references instead